FirebaseCore.podspec

```diff
@@ -48,4 +48,12 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
     unit_tests.dependency 'OCMock'
     unit_tests.resources = 'Example/Core/App/GoogleService-Info.plist'
   end
+
+  s.test_spec 'swift-unit' do |swift_unit_tests|
+    swift_unit_tests.source_files = 'Example/Core/Tests/Swift/**/*.swift',
+                                    'Example/Core/Tests/Swift/**/*.h'
+    swift_unit_tests.pod_target_xcconfig = {
+      'SWIFT_OBJC_BRIDGING_HEADER' => '$(PODS_ROOT)/../../Example/Core/Tests/Swift/FirebaseCore-iOS-Unit-swift-unit-Bridging-Header.h'
```

Comment:
Should probably be `PODS_TARGET_SRCROOT`.

Reply:
Great, that works! Thank you for the suggestion.